BAZProject/MovieDetail/MovieDetailPresenter.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  MovieDetailPresenter.swift
+//  BAZProject
+//
+//  Created 1030364 on 17/10/22.
+//  Copyright © 2022 ___ORGANIZATIONNAME___. All rights reserved.
+//
+//  Template generated by Juanpe Catalán @JuanpeCMiOS
+//
+
+import UIKit
+
+class MovieDetailPresenter: MovieDetailPresenterProtocol {
```

Comment:
Agrega `final` a tus clases 
```suggestion
final class MovieDetailPresenter: MovieDetailPresenterProtocol {
```